### DISCUSSION TITLE
Disable Dark Theme and Get Rid of Custom Properties

### DISCUSSION THREAD

**Discussion by mostafaznv** [vuetifyjs/vuetify#17699] [Q&A]
Hi everyone.

I have two questions:
1. How can I disable the dark theme? I don't need it and I don't want to see all its CSS selectors and custom properties in my final build.

2. How can I get rid of the custom properties and the `<style id="vuetify-theme-stylesheet">` tag in my web pages? A workaround for this is to disable the theme completely, but I want to use the theme functionality (only to set colors for the light theme and use them throughout the app).

```html
<!-- I dont want these custom propeties !-->

<style id="vuetify-theme-stylesheet">
:root {
    --v-theme-background: 255,255,255;
    --v-theme-background-overlay-multiplier: 1;
    --v-theme-surface: 255,255,255;
    --v-theme-surface-overlay-multiplier: 1;
    --v-theme-surface-variant: 66,66,66;
    --v-theme-surface-variant-overlay-multiplier: 2;
    --v-theme-on-surface-variant: 238,238,238;
...
</style>
```

**Comment by MoeinMirkiani** [comment]
I'm having struggling with the same issues :(
